tg/views/__init__.py

```diff
@@ -256,17 +289,20 @@ def _collect_msgs_to_draw(
             line_num = self.h
             for msg_idx, msg_item in msgs[ignore_before:]:
                 is_selected_msg = current_msg_idx == msg_idx
-                dt, user_id_item, msg = self._parse_msg(msg_item)
+                msg_proxy = MsgProxy(msg_item)
+                dt = msg_proxy.date.strftime("%H:%M:%S")
+                user_id_item = msg_proxy.sender_id
+
+                msg, is_reply = self._format_msg(msg_proxy, user_id_item)
                 user_id = self._get_user_by_id(user_id_item)
-                msg = msg.replace("\n", " ")
+
                 # count wide character utf-8 symbols that take > 1 bytes to
                 # print it causes invalid offset
                 wide_char_len = sum(map(len, emoji_pattern.findall(msg)))
-                elements = (f" {dt} ", user_id, " " + msg)
-                # elements = tuple(map(lambda x: f" {x}", (dt, user_id, msg)))
+                elements = tuple(map(lambda x: f" {x}", (dt, user_id, msg)))
```

Comment:
don't do it, I intentionally make padding with my hands to make it look good

Reply:
fixed